src/main/webapp/WEB-INF/tags/instructor/instructorSearch/commentsForStudentsSearchResults.tag

```diff
@@ -0,0 +1,45 @@
+<%@ tag description="instructorSearch.jsp - Search comments for students" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<br>
+<div class="panel panel-primary">
+    <div class="panel-heading">
+        <strong><jsp:doBody></jsp:doBody></strong>
+    </div>
+                
+    <div class="panel-body">
+        <c:set var="commentIdx" value="${0}"/>
```

Comment:
`varStatus` here too.
By the way, you can use a self-closing doBody tag (`<jsp:doBody />`) instead of that.